CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+* Released version is no published as a pact as it was inaccurate
```

Comment:
```suggestion
* Removes publishing pacts for released versions as they incorrectly referenced unreleased commits.
```

Also don't know if this should be in the change log, as it's not a user facing change?

Reply:
Argh my English 🙈 thanks, I've updated.

It's somewhat moot since no-one uses them but I figured since it was something public that is consumed by consumers then it counts as a user land change